pages/examples/stopwatch.py

```diff
@@ -0,0 +1,62 @@
+import dash
+from dash.dependencies import Input, Output
+import dash_daq as daq
+import dash_core_components as dcc
+import dash_html_components as html
+import time 
+
+app = dash.Dash(__name__)
+
```

Comment:
@someshfengde 
I see this is still not working. What if you try something like this code? Is that what you're looking for?

```
app.layout = html.Div([

    daq.LEDDisplay(
        id='stopwatch-y',
        label="Default",
        value=6
    ),
    dcc.Input(
        id="stopwatch-input",
        type="number",
        value=1,
        placeholder="input type integer",
    ),
    daq.PowerButton(
        id='stopwatch-start',
        on=False,
        color='#FF5E5E',
        theme="dark"
    ),
    dcc.Interval(id='trigger-while-button-on',
                 interval=1*1000,
                 n_intervals=0)
])


@app.callback(
    Output('stopwatch-y', 'value'),
    Input('trigger-while-button-on', 'n_intervals'),
    Input("stopwatch-start", 'on'),
    Input('stopwatch-y', 'value'),
)
def update_output(n, start_watch, led_value):

    print(n)
    if n > 0 and start_watch == True:
        if led_value > 0:
            led_value -= 1
            return led_value

    return led_value


# @app.callback(
#     Output('stopwatch-y', 'value'),
#     Input('stopwatch-start', 'value')
# )
# def update_time(value):
#     return str(value)

if __name__ == '__main__':
    app.run_server(debug=True)

```

Reply:
I didn't use the input component but you can probably incorporate it if you like this solution.

Reply:
I've tried same with input as number it's not working